src/core_modules/capture-core/components/TemplateSelector/TemplateSelector.component.tsx

```diff
@@ -1,17 +1,15 @@
-// @flow
-import React, { useMemo, useState } from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import React, { useMemo, useState, type ComponentType, type ReactNode } from 'react';
+import { withStyles, createStyles, type WithStyles } from '@material-ui/core/styles';
 import cx from 'classnames';
 import { Button, spacers, colors } from '@dhis2/ui';
 import i18n from '@dhis2/d2-i18n';
+import { BookmarkAddIcon } from 'capture-ui/Icons';
 import { localeCompareStrings } from '../../utils/localeCompareStrings';
+import { Widget } from '../Widget';
 import { TemplateSelectorChip } from './TemplateSelectorChip.component';
 import type { WorkingListTemplates, WorkingListTemplate } from './workingListsBase.types';
-import { Widget } from '../Widget';
-import { BookmarkAddIcon } from '../../../capture-ui/Icons';
-
 
-const getStyles = () => ({
```

Comment:
I didn't know about this utility function. This is a lot better than adding the `as ComponentType<Props>` to the export. Can you add that to your knowledge bank, so that we keep it in mind in the future?

Reply:
Did you add this to your knowledge bank?